cmd/cloud/installation_flag.go

```diff
@@ -78,11 +78,13 @@ func (flags *installationCreateFlags) addFlags(command *cobra.Command) {
 }
 
 type installationPatchRequestChanges struct {
-	ownerIDChanged bool
-	versionCHanged bool
-	imageChanged   bool
-	sizeChanged    bool
-	licenseChanged bool
+	ownerIDChanged        bool
+	versionCHanged        bool
+	imageChanged          bool
+	sizeChanged           bool
+	licenseChanged        bool
+	allowedRangesChanged  bool
```

Comment:
I suggest making this a bit more specific like `allowedIPRanges` so it's easier to understand at a glance (I'd say the same for anywhere else like `overrideIPRangesChanged`, `internalIPRanges` etc.